package.json

```diff
@@ -13,11 +13,14 @@
     "clean": "rimraf lib && rimraf coverage",
     "lint": "tsc --noEmit && tslint --project tsconfig.json",
     "prebuildOnly": "yarn clean && yarn format && yarn lint && echo Using TypeScript && tsc --version",
-    "build": "tsc --pretty",
+    "build": "rimraf lib && tsc --pretty",
     "test": "jest",
     "coverage": "jest --coverage",
     "watch": "yarn build --watch",
-    "watch:test": "jest --watch"
+    "watch:test": "jest --watch",
```

Comment:
not sure why these changes are changes ... they are already there ?

Reply:
comma change

Reply:
also a local unpushed merge ... sigh.
Not my day for Node.